README.md

```diff
@@ -817,27 +817,20 @@ customButtons: [{
 
 - **customButtons.click** - Function that is called when the button is clicked
 
-## Adding Registration Link
+## Adding Registration Button
 
-You can add registration link underneath the login form on the primary auth page by setting the following config options. If you'd like to change the label text and link text use the `i18n` config option.
+You can add a registration button to the primary auth page by setting the following config options. If you'd like to change the text, use the i18n config option
```

Comment:
I'll add that back . I was wondering if there is an easy way to check how the read me formatting looks locally before pushing ?